### DISCUSSION TITLE
Fastify as daemon on AWS Lightail

### DISCUSSION THREAD

**Discussion by smuralimohan** [fastify/fastify#3584] [Q&A] [answered]
Hi,

I am trying to deploy Fastify on AWS Lightsail. I tried the following command to run Fastify as a background process,

_**nohup fastify start dist/app.js &**_

,but when I close the Lightsail terminal, the process gets killed and the fastify stops running.  Are there some ways to deploy Fastify on AWS Lightsail and make it run as a daemon? 

Help appreciated. 

Thank you,
Murali

**Comment by kamikazechaser** [comment, marked as answer]
A better approach is to use a systemd user service or docker. The daemon should restart across reboots. nohup or tmux aren't reliable ways of "daemonizing" application servers.

**Reply by smuralimohan** [reply]
Hey, thanks @kamikazechaser . You are right . I too felt the same after posting the question.
